### ISSUE TITLE
Show added/removed lines column

### ISSUE THREAD

**Issue by GiovanniCardamone** [mhutchie/vscode-git-graph#705]
Would be nice to have an additional column that summarize how many lines has been added/removed in commit entry

![Screenshot from 2022-09-27 15-34-22](https://user-images.githubusercontent.com/5117748/192541410-d22fad81-5ab8-4596-8fe0-a7534df2a623.png)

**Comment by dimateos** [comment]
Yeah I miss this column too. Pretty useful to differentiate small/large commits.

Sample design from git log:
![image](https://github.com/mhutchie/vscode-git-graph/assets/26868611/8fd37c73-5642-4cba-b830-246fbcc08332)

Sample design from gitlens:
![image](https://github.com/mhutchie/vscode-git-graph/assets/26868611/dc044ec3-f466-4abd-9d06-2bebf501875f)

Could be as simple as additions in green, deletions in red, gray for file count
